app/auth/google_auth.py

```
@@ -0,0 +1,73 @@
+import aiohttp
+import requests
```

Ruff (0.8.2)

`requests` imported but unused

Remove unused import: `requests`

(F401)